profiles/forms.py

```diff
@@ -30,3 +33,61 @@ def __init__(self, *args: tuple, **kwargs: dict) -> None:
             ),
         )
         self.field_order = ["first_name", "last_name", "bio", "birth_date"]
+
+
+class EditProfileLinksForm(forms.ModelForm):
+    github_link = forms.CharField(
+        label="",
+        required=False,
+        widget=forms.TextInput(
+            attrs={"placeholder": "GitHub", "class": "col s12 social_links"}
+        ),
+    )
+    linkedin_link = forms.CharField(
+        label="",
+        required=False,
+        widget=forms.TextInput(
+            attrs={"placeholder": "Linkedin", "class": "col s12 social_links"}
+        ),
+    )
+    instagram_link = forms.CharField(
+        label="",
+        required=False,
+        widget=forms.TextInput(
+            attrs={"placeholder": "Instagram", "class": "col s12 social_links"}
+        ),
+    )
+
+    class Meta:
+        model = Profile
+        fields = ["github_link", "linkedin_link", "instagram_link"]
+
+    def github(self) -> str:
+        github = self.cleaned_data.get("github_link")
+
+        if github == "":
+            return ""
+        elif github.startswith("https://github.com/"):
+            return github
+        else:
+            return "Error"
```

Comment:
I think there should be something like
```suggestion
            raise ValidationError('github url is not valid')
```

Reply:
and the same for others

Reply:
raise ValidationError('github url is not valid') returns error on backend
![Screenshot 2024-03-22 at 19 44 17](https://github.com/lambda-py/py_june/assets/48783453/ca60517a-4d0e-4e62-911f-16c17a622091)